uwblueprint/childrens-aid-society
# Kevin y/change status modal status dropdown

## Notion ticket link

[Change Status Modal Status Dropdown](https://www.notion.so/uwblueprintexecs/134e1f159caa4f73939d11a700c325ad?v=39cf7d5a98384eb9af2bff2265d0ac8a&p=ad66a5760b6d4525856992fe46bc01f3&pm=s)



## Implementation description
* Used existing custom tag with popovers and custom input to replicate a custom dropdown select.
* Used InputLeftAddon to display the tag in the input and a map of colors for each tag
![image](https://github.com/uwblueprint/childrens-aid-society/assets/70108329/9e86ecac-4bfd-46a4-8a2b-1c9532002f39)



## Steps to test
1. Selecting dropdown options should update state and change what the input displays
2. The colors match the design



## What should reviewers focus on?
* 


## Checklist
